jujugui/static/gui/src/app/init/component-renderers-mixin.js

```diff
@@ -342,6 +343,56 @@ const ComponentRenderersMixin = (superclass) => class extends superclass {
   }
 
   /**
+    Display post deployment help.
+
+    @param {Object} state The entity ID of the charm or bundle.
+    @param {Function} next Run the next handler.
+  */
+  _displayPostDeployment(state, next) {
+    if (!state.postDeploymentPanel.show) {
+      next();
+      return;
+    }
+    const entityId = state.postDeploymentPanel.entityId;
+    this.state.changeState({
+      'postDeploymentPanel': null
```

Comment:
quotes aren't necessary here.

So from within the state callback for the post deployment you then want to clear it? Shouldn't it only be cleared when they hit X?

Reply:
done